### PR TITLE
wofi: add page

### PR DESCRIPTION
**Pull request by MuhammadAly11**
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** v1.4.1

### CODE REVIEW

File: pages/linux/wofi.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# wofi
+
+> An application launcher for wayland, similar to rofi and dmenu.
```
Comment (Managor) [inline pages/linux/wofi.md:3]: I'm assuming this isn't a universal command.
```suggestion
> An application launcher for wlroots-based wayland compositors, similar to rofi and dmenu.
```

**Review by Managor — APPROVED** [review]

File: pages/linux/wofi.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`wofi --show drun`
+
+- Show the list of all commands:
+
+`wofi --show run`
+
+- Pipe a list of items to `stdin` and print the selected item to `stdout`:
+
+`printf "Choice1\nChoice2\nChoice3" | wofi --dmenu`
```
Comment (spageektti) [inline pages/linux/wofi.md:16]:
```suggestion
`printf "{{Choice1\nChoice2\nChoice3}}" | wofi --dmenu`
```

File: pages/linux/wofi.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# wofi
+
+> An application launcher for wlroots-based wayland compositors, similar to rofi and dmenu.
```
Comment (acuteenvy) [inline pages/linux/wofi.md:3]:
```suggestion
> An application launcher for wlroots-based Wayland compositors, similar to `rofi` and `dmenu`.
```

**Review by acuteenvy — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Comment by spageektti** [comment]
Thanks!

**Pull request merged**